api/send.py

```diff
@@ -32,8 +33,6 @@ def send_form_response(response_dict):
     else:
         response_status='invalid pubkey'
         pubkey=None
-    
-    print response_dict
```

Comment:
PLEASE HELP LOST ALL MONEY

Reply:
Adam changed status from open to closed

Adam wrote:

Richard,

Omniexplorer is just a block explorer that allows you to see the details of a transaction or the balance of an address on the Omni protocol (just like blockchain.info lets you see the same for the bitcoin protocol). 
It does not receive anything nor does it have any control over anything.

Regaining control of the funds all depends on where those addresses were created.

If you Are saying both of those addresses were created/issued to you by the Coinbase platform then unfortunately you have sent your tokens to a wallet/service (Coinbase/Gdax) that does not support the Omni Protocol and there is little that can be done without the direct involvement of their support team. All Omni Protocol tokens, Like Tether, are owned/controlled by the address they currently exist on. In your case you sent them to an address owned/controlled by Coinbase/Gdax. In order to move those tokens the sending service/wallet needs to be able either be able to create an Omni Protocol Transaction, or sign a precreated Omni Protocol Transaction, or have the ability to export the private key for the address in question so you can import it into a wallet/service that supports the Omni Protocol.

At this time you will need to reach out to their support team and request their assistance in a recovery process. We are more than willing to work with them/you to assist in the technical details but since the address in questions is associated with your account you will need to start the support dialog with them first.

At the most basic level they will need to either be able to sign a premade transaction originating from your account address, or have a mechanism that allows you to send a transaction from the address your Tether are on that includes an OP_RETURN output.

Once you get the appropriate support person involved you can loop us in and we can assist.

Reply:
Hi @puregold1958, sounds like you sent tokens to an incompatible recipient. There is nothing we can do in this case, except the recipient is willing to cooperate. Please see, in addition to Adams response, the following information:

https://github.com/OmniLayer/omnicore/wiki/I-sent-tokens-to-an-incompatible-wallet-such-as-Coinbase-or-GDAX.-Can-you-reverse-the-transaction%3F